packages/firestore/src/api/credentials.ts

```diff
@@ -151,6 +152,7 @@ export class FirebaseCredentialsProvider implements CredentialsProvider {
     this.tokenListener = () => {
       this.tokenCounter++;
       this.currentUser = this.getUser();
+      this.receivedInitialUser = true;
```

Comment:
Could we just add a `this.currentUser = this.getUser();` call below so that it gets initialized in the constructor?  This would match Android (https://github.com/firebase/firebase-android-sdk/blob/29a1573ac1c6b7765aa6ed95ceafc95ec3b670f0/firebase-firestore/src/main/java/com/google/firebase/firestore/auth/FirebaseAuthCredentialsProvider.java#L73)

Reply:
Good call; done.